lesson03/assignment/css/_components.content.scss

```diff
@@ -13,5 +13,52 @@ html {
 }
 
 h1 {
-    color: $red;  
-}
\ No newline at end of file
+    color: $red;
+}
+
+.template-hook{
+	display: flex;
+	flex-flow: row wrap;
+	height: auto;
+	.employee_cards{
+		display:flex;
+		justify-content: center;
+		flex-direction: column;
+		margin: 0  auto;
+	}
+	h1{
+		margin: 2% 0;
+		text-align: center;
+	}
+}
+
+//Info about employee: Nesting
```

Comment:
You've got your nesting down, however the goal of assignment2/3 was to build the component as designed, or at least as close as you can. Were you able to use XD or the design spec link to grab the styling details? Please take a look at reworking your CSS to fit the design.